### PR TITLE
Hw6

### PR DESCRIPTION
**Pull request by EvgeniyTT**

### CODE REVIEW

File: dbp/connect.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Sequelize from 'sequelize';
+
+const sequelize = new Sequelize('postgres://postgres:postgres@postgres:5432/postgres');
+
+export const connectToPostgresDb = () => {
+  sequelize
```
Comment (thebergamo) [inline dbp/connect.js]: why not async/await here as well?

**Comment by thebergamo** [comment]
Things looks ok, Would just adding few tips for you:

- I would abstract more routes to use `controllers` instead of implementing those directly. Even if it is really simple function, having this abstraction you are able to migrate and change your http server any time without lots of rewriting.
- There is a concept called migrations and seeds while talking with SQL databases that would be nice to you to have it. Migrations are basically a descriptions about how your tables are evolving and so on. (http://docs.sequelizejs.com/manual/tutorial/migrations.html)
- For MongoDB you don't need to the concept of migration because of the schemaless stuff, but seeds would help to not fill your database every time you start your server. Sequelize provides a way to do it, but Mongoose don't out of the box, but you can just get your `fillUpMongo` function and run it as just a script. :D 
- Also, unless it's just for your organization, having routes translated as MongoRoutes or anything similar, it would be not good for a real life project, usually this stuff are abstracted and while developing would be better treat all your models, routes or controllers more equally as possible.

File: dbp/connect.js (hunk 1 above)
Comment (EvgeniyTT) [reply dbp/connect.js]: Good point. Because of copy-paste :) Changed.

**Comment by EvgeniyTT** [comment]
Hi @thebergamo , thank you so much for detailed review and comments. I made changes based on them, except the migration. I read about it, it looks cool. Do you want me to implement it in the scope of this PR or it's more just for knowledge that such things exist?

**Comment by thebergamo** [comment]
Hey, I'm happy that my review was useful for you :)

About migrations as you wish, having those in mind will help you in many ways to keep your schemas in a more controllable way for sure :)

**Pull request merged**